### ISSUE TITLE
[tsconfig] Wrong error on compiler option 'paths'

### ISSUE THREAD

**Issue by RKaczmarek** [Everlaw/nbts#87]
Hi,

the tsconfig parser shows an error on the option "paths": "Compiler option 'paths' requires a value of type object."

But the value for this option is an object:

```json
{
    "name": "any",
    "compilerOptions": {
        "baseUrl": ".",
        "paths": {
            "anymodule": ["any/path"],
            "othermodule": ["other/path"]
        }
    }
}
```

**Comment by dylanvorster** [comment]
Same issue here.

**Comment by jeffrey-easyesi** [comment]
Fixed in v2.1.5.2
